### PR TITLE
Breadcrumbs pattern examples 

### PR DESCRIPTION
**Pull request by JoanaMMoreira**

### CODE REVIEW

File: .changeset/khaki-eggs-brake.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"@salt-ds/core": patch
+---
+
+Added storybook examples for the breadcrumbs pattern
```
Comment (joshwooding) [inline .changeset/khaki-eggs-brake.md:5]: We don't need a changeset for pattern examples :)

File: packages/core/stories/patterns/breadcrumbs/breadcrumbs.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+const Separator = () => (
+  <li aria-hidden style={{ display: "flex" }}>
+    <ChevronRightIcon />
+  </li>
+);
+
+export const Breadcrumbs = () => {
+  return (
+    <nav aria-label="Breadcrumb">
+      <ol
```
Comment (joshwooding) [inline packages/core/stories/patterns/breadcrumbs/breadcrumbs.stories.tsx:20]: ul might be better

File: packages/core/stories/patterns/breadcrumbs/breadcrumbs.stories.tsx [hunk 3]
```diff
@@ -0,0 +1,196 @@
...
+          columnGap: "var(--salt-size-unit)",
+          listStyle: "none",
+          flexWrap: "wrap",
+        }}
+      >
+        <li>
+          <Link href="#">Home</Link>
+        </li>
+        <Separator />
+        <MenuButton CascadingMenuProps={{ initialSource }} hideCaret>
```
Comment (joshwooding) [inline packages/core/stories/patterns/breadcrumbs/breadcrumbs.stories.tsx:133]: This should be swapped to Menu in core

**Review by joshwooding — CHANGES_REQUESTED** [review]

**Review by joshwooding — CHANGES_REQUESTED** [review]
The site needs to be updated with a link to the examples

**Review by joshwooding — APPROVED** [review]

**Review by Fercas123 — APPROVED** [review]

**Pull request merged**